Libraries/LibDebug/Dwarf/DwarfInfo.h

```diff
@@ -51,12 +52,12 @@ class DwarfInfo : public RefCounted<DwarfInfo> {
     explicit DwarfInfo(NonnullRefPtr<const ELF::Loader> elf);
     void populate_compilation_units();
 
-    ByteBuffer section_data(const String& section_name);
+    Optional<ByteBuffer> section_data(const String& section_name);
 
     NonnullRefPtr<const ELF::Loader> m_elf;
-    ByteBuffer m_debug_info_data;
-    ByteBuffer m_abbreviation_data;
-    ByteBuffer m_debug_strings_data;
+    Optional<ByteBuffer> m_debug_info_data;
+    Optional<ByteBuffer> m_abbreviation_data;
+    Optional<ByteBuffer> m_debug_strings_data;
```

Comment:
`ByteBuffer` already has a null state (which is what you get with `ByteBuffer()`).
Let's use that instead of wrapping them in `Optional` :)